packages/web/integrations/notifi/notifi-context.tsx

```diff
@@ -75,18 +77,20 @@ const NotifiContextProviderImpl: FunctionComponent<PropsWithChildren<{}>> =
           return Buffer.from(result.signature, "base64");
         }}
       >
-        <NotifiConfigContext
-          type="SUBSCRIPTION_CARD"
-          id={
-            IS_TESTNET
-              ? "c5fb30811f5b47b79dd9a400480c2670"
-              : "c5fb30811f5b47b79dd9a400480c2670"
-          }
-        >
-          <NotifiModalContextProvider account={info.account.address}>
-            {children}
-          </NotifiModalContextProvider>
-        </NotifiConfigContext>
+        <NotifiLocalStorageProvider>
+          <NotifiConfigContext
+            type="SUBSCRIPTION_CARD"
+            id={
+              IS_TESTNET
+                ? "c5fb30811f5b47b79dd9a400480c2670"
+                : "c5fb30811f5b47b79dd9a400480c2670"
```

Comment:
Heads up these appear to be the same

Reply:
Yah, both Dev and Prod share the same cardId. If it is confusing, I will remove the condition check